packages/compass/src/main/telemetry.ts

```diff
@@ -17,7 +17,7 @@ const SEGMENT_API_KEY =
 const SEGMENT_HOST =
   process.env.HADRON_METRICS_SEGMENT_HOST_OVERRIDE ||
   process.env.HADRON_METRICS_SEGMENT_HOST;
-const IS_CI = process.env.IS_CI;
+const IS_CI = process.env.IS_CI || process.env.CI || process.env.EVERGREEN_BUILD_VARIANT;
```

Comment:
This is just me being a bit more careful, I think we have `IS_CI` already in evergreen but this way we can be sure that this is not reporting telemetry for our e2e tests :)